docs/man/docker-cp.1.md

```diff
@@ -11,8 +12,9 @@ CONTAINER:PATH HOSTPATH
 
 # DESCRIPTION
 Copy files/folders from a container's filesystem to the host
-path. Paths are relative to the root of the filesystem. Files
-can be copied from a running or stopped container.
+path running the command. Paths are relative to the root of
```

Comment:
only the path in the container is relative to the root - the HOSTPATH is relative to the current dir (unless it starts with /)